<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1477902987391" stop="1477903095731" version="1.4.22" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default Suite : test</name>
    <title>Default Suite : test</title>
    <test-cases>
        <test-case start="1477902994851" stop="1477903029558" status="passed">
            <name>before_test</name>
            <description>Удаление тестовых данных.</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903029621" stop="1477903042648" status="passed">
            <name>step1_openHospitalModule</name>
            <description>Открытие модуля 'Диспансеризация детей сирот'.</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903042653" stop="1477903044404" status="passed">
            <name>step2_ChooseEvent</name>
            <description>Выбор в дереве мероприятия 'ОДОДСГ'.</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903044409" stop="1477903047084" status="passed">
            <name>step3_ClickAddPatientButtonAndCheckModal</name>
            <description>Нажать кнопку 'Добавить пациента'. Проверить, что появилось модальное окно. Проверить, что в поле 'Дата выдачи бланка' заполнено текущее число.</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903047090" stop="1477903048965" status="passed">
            <name>step4_openPatientsSearchModal</name>
            <description>Открытие модального окна 'Поиск пациентов'.</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903048969" stop="1477903063907" status="passed">
            <name>step5_fillFieldsInPatientsSearchModal</name>
            <description>Заполнение полей в модальном окне 'Поиск пациентов'.</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903063911" stop="1477903066744" status="passed">
            <name>step6_checkFoundedRecord</name>
            <description>Выберите найденного пациента и нажмите кнопку 'Выбрать'. Проверьте, что в модальном окне  в поле 'Пациент' отобразился выбранный нами пациент.</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903066748" stop="1477903068178" status="passed">
            <name>step7_fillFieldsAndCheckSuccessSaving</name>
            <description>Заполнение полей на форме в модальном окне 'Добавить пациента', проверка успешного добавления.</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903068181" stop="1477903082486" status="passed">
            <name>step8_checkDataInCardPage</name>
            <description>Выбор мероприятия. Выбор записи в таблице, нажатие кнопки 'Карточка' и проверка данных в контейнере 'Общие сведения'.</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903082490" stop="1477903085444" status="passed">
            <name>step9_selectRecordInServiceTable_AndClickInspectButton</name>
            <description>Выбрать первую запись в таблице 'Осмотры и услуги' и нажать кнопку 'Осмотр'</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903085447" stop="1477903095725" status="failed">
            <name>step10_executeExamination</name>
            <description>Проверка, что открылось модальное окно 'Оказание услуги' и произвести выполнение осмотра.</description>
            <failure>
                <message>AssertionError: Success alert message is not present
Expected: is &lt;true&gt;
     but: was &lt;false&gt;</message>
                <stack-trace>java.lang.AssertionError: Success alert message is not present
Expected: is &lt;true&gt;
     but: was &lt;false&gt;
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at net.framework.autotesting.meta.Alerts.assertSuccessAlertIsPresent(Alerts.java:45)
	at businessFramework.module.dispensary.pages.executeExamination.ExecuteExaminationPage.executeExamination(ExecuteExaminationPage.java:18)
	at test.businessTests.dispensary.DispensaryPatientTest.step10_executeExamination(DispensaryPatientTest.java:134)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:124)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903095726" stop="1477903095727" status="canceled">
            <name>step11_deleteDiagnosis</name>
            <description>Удаление созданного диагноза.</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:356)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:171)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1887)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:124)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1477903095728" stop="1477903095728" status="canceled">
            <name>step12_deletePatient</name>
            <description>Удаление пациента из мероприятия.</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:356)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:171)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1887)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:124)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="5920@KZN-062.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
